src/temporal/client.ts

```diff
@@ -0,0 +1,19 @@
+import { Client } from '@temporalio/client';
+import { Connection as TemporalConnection } from '@temporalio/client/lib/connection';
+import { TEMPORAL_ADDRESS, TEMPORAL_NAMESPACE } from './config';
+
+let client: Client;
+
+export const getTemporalClient = async (): Promise<Client> => {
+  if (client) {
+    return client;
+  }
+
+  const connection = await TemporalConnection.connect({
+    address: TEMPORAL_ADDRESS,
```

Comment:
This is still not production ready right?

Reply:
Yep, this will completely be different based on what I saw from `gatekeeper`.

Reply:
Connected with Denis, and as discussed, the setup for client on how they do it now is the same as the one in our `gatekeeper` repo. I've now updated how we fetch the options for connecting to either locally or to prod.